python/django/use-decimalfield-for-money.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: use-decimalfield-for-money
+    patterns:
+      - pattern-inside: |
+          class $M(...):
+            ...
+      - pattern: $F = django.db.models.FloatField(...)
+      - pattern-where-python: "'price' in vars['$F'] or 'amount' in vars['$F'] or 'subtotal' in vars['$F'] or 'donation' in vars['$F'] or 'fee' in vars['$F'] or 'salaray' in vars['$F'] or 'precio' in vars['$F']"
```

Comment:
@minusworld @ulziibay I think there is a typo here. `salaray' in vars['$F']` should have been `salary' in vars['$F']`

Reply:
🙊 Yup, good catch!